pvlib/pvsystem.py

```diff
@@ -635,52 +638,59 @@ def sapm(module, poa_direct, poa_diffuse, temp_cell, airmass_absolute, aoi):
 
     Notes
     -----
-    The coefficients from SAPM which are required in ``module`` are:
-
-    ========   ===============================================================
-    Key        Description
-    ========   ===============================================================
-    A0-A4      The airmass coefficients used in calculating
-               effective irradiance
-    B0-B5      The angle of incidence coefficients used in calculating
-               effective irradiance
-    C0-C7      The empirically determined coefficients relating
-               Imp, Vmp, Ix, and Ixx to effective irradiance
-    Isco       Short circuit current at reference condition (amps)
-    Impo       Maximum power current at reference condition (amps)
-    Aisc       Short circuit current temperature coefficient at
-               reference condition (1/C)
-    Aimp       Maximum power current temperature coefficient at
-               reference condition (1/C)
-    Bvoco      Open circuit voltage temperature coefficient at
-               reference condition (V/C)
-    Mbvoc      Coefficient providing the irradiance dependence for the BetaVoc
-               temperature coefficient at reference irradiance (V/C)
-    Bvmpo      Maximum power voltage temperature coefficient at
-               reference condition
-    Mbvmp      Coefficient providing the irradiance dependence for the
-               BetaVmp temperature coefficient at reference irradiance (V/C)
-    N          Empirically determined "diode factor" (dimensionless)
+    The coefficients from SAPM which are required in ``module`` are listed in
+    the following table.
+
+    Remark that modules of the Sandia module database contain these
+    coefficients but modules of the CEC module database do not. Both databases
+    can be accessed using :py:func:`retrieve_sam`.
+
```

Comment:
Minor suggested edit...

The modules in the Sandia module database contain these coefficients, but the modules in the CEC module database do not.

Or you could replace "Remark that" with something like "We note that".